spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -63,6 +63,51 @@
         before do
           request.headers['Authorization'] = socure_secret_key
         end
+
+        shared_examples 'repeats webhooks' do
+          let(:headers) do
+            {
+              Authorization: socure_secret_key_queue.last,
+              'Content-Type': 'application/json',
+            }
+          end
+          let(:endpoints) do
+            3.times.map { |i| "https://#{i}.example.test/endpoint" }
+          end
+          let(:repeated_body) do
+            b = {}.merge(webhook_body)
+            b[:event].delete(:data)
+            b
+          end
+
+          before do
+            allow(IdentityConfig.store)
+              .to receive(:socure_docv_webhook_repeat_endpoints).and_return(endpoints)
+            allow(SocureDocvResultsJob).to receive(:perform_now)
+            allow(SocureDocvRepeatWebhookJob).to receive(:perform_later)
+
+            dcs = create(:document_capture_session, :socure)
+            webhook_body[:event][:docvTransactionToken] = dcs.socure_docv_transaction_token
+
+            request.headers['Authorization'] = headers[:Authorization]
+            request.headers['Content-Type'] = headers[:'Content-Type']
+
+            endpoints.each do |endpoint|
+              expect(SocureDocvRepeatWebhookJob).to receive(:perform_later) do |*args|
+                expect(args.first[:body]).to eq(JSON.parse(repeated_body.to_json))
+                expect(args.first[:endpoint]).to eq(endpoint)
+                expect(args.first[:headers]).to eq(headers)
+              end
+            end
+          end
+
+          context 'when idv workers are enabled' do
+            it 'queues SocureDocvRepeatWebhook jobs' do
+              post :create, params: webhook_body
+            end
+          end
```

Comment:
If we move the expects inside it `it` it's more obvious what is actually being tested:
```suggestion
          end

          context 'when idv workers are enabled' do
            it 'queues SocureDocvRepeatWebhook jobs' do
              endpoints.each do |endpoint|
                expect(SocureDocvRepeatWebhookJob).to receive(:perform_later) do |*args|
                  expect(args.first[:body]).to eq(JSON.parse(repeated_body.to_json))
                  expect(args.first[:endpoint]).to eq(endpoint)
                  expect(args.first[:headers]).to eq(headers)
                end
              end

              post :create, params: webhook_body
            end
          end
```